module/data/item/class.mjs

```diff
@@ -0,0 +1,79 @@
+import SystemDataModel from "../abstract.mjs";
+import { AdvancementField, IdentifierField } from "../fields.mjs";
+import ItemDescriptionTemplate from "./templates/item-description.mjs";
+
+/**
+ * Data definition for Class items.
+ * @mixes ItemDescriptionTemplate
+ *
+ * @property {string} identifier        Identifier slug for this class.
+ * @property {number} levels            Current number of levels in this class.
+ * @property {string} hitDice           Denomination of hit dice available as defined in `DND5E.hitDieTypes`.
+ * @property {number} hitDiceUsed       Number of hit dice consumed.
+ * @property {object[]} advancement     Advancement objects for this class.
+ * @property {string[]} saves           Savings throws in which this class grants proficiency.
+ * @property {object} skills            Available class skills and selected skills.
+ * @property {number} skills.number     Number of skills selectable by the player.
+ * @property {string[]} skills.choices  List of skill keys that are valid to be chosen.
+ * @property {string[]} skills.value    List of skill keys the player has chosen.
+ * @property {object} spellcasting      Details on class's spellcasting ability.
+ * @property {string} spellcasting.progression  Spell progression granted by class as from `DND5E.spellProgression`.
+ * @property {string} spellcasting.ability      Ability score to use for spellcasting.
+ */
+export default class ClassData extends SystemDataModel.mixin(ItemDescriptionTemplate) {
+  static defineSchema() {
+    return this.mergeSchema(super.defineSchema(), {
+      identifier: new IdentifierField({required: true, label: "DND5E.Identifier"}),
+      levels: new foundry.data.fields.NumberField({
+        required: true, nullable: false, integer: true, positive: true, initial: 1, label: "DND5E.ClassLevels"
+      }),
+      hitDice: new foundry.data.fields.StringField({required: true, initial: "d6", label: "DND5E.HitDice"}),
```

Comment:
```suggestion
      hitDice: new foundry.data.fields.StringField({required: true, initial: "d6", blank: false, label: "DND5E.HitDice"}),
```

We can also add a `validate` here to check for the correct format (i.e. `/d\d+/`), since I believe we make some assumptions about this elsewhere in the codebase.